scripts/linters/pre_commit_linter.py

```diff
@@ -230,13 +230,15 @@ def _lint_all_files(
     css_file_extension_type = 'css' in file_extensions_to_lint
     other_file_extension_type = 'other' in file_extensions_to_lint
 
-    if not other_file_extension_type:
-        general_files_to_lint = '.' + file_extensions_to_lint
+    if js_ts_file_extension_type:
+        general_files_to_lint = _FILES['.js'] + _FILES['.ts']
```

Comment:
Assume someone passes `ts` as a param for file extension to lint then will this script works as expected? (i.e, only lint .ts files.)

Reply:
This will lint 'js' files as well and I think it is fine as we do not have many 'js' files. So it does not matter much.

Reply:
But this isn't what an end-user will expect while running the test, right? I would suggest either find a way to lint only the files with a given extension or update the help section of the argparse with this info.

Reply:
Update argparse help section.